ecs-cli/modules/cli/compose/project/project_parseV3.go

```diff
@@ -192,3 +193,19 @@ func convertPortConfigToECSMapping(portConfig types.ServicePortConfig) *ecs.Port
 	}
 	return &ecsMapping
 }
+
+func logWarningForDeployFields(d types.DeployConfig, serviceName string) {
+	if d.Resources.Limits != nil || d.Resources.Reservations != nil {
+		log.WithFields(log.Fields{
+			"option name":  "deploy",
+			"service name": serviceName,
+		}).Warn("Skipping unsupported YAML option for service... service-level resources should be configured in the ecs-param.yml file.")
+	} else if d.Mode != "" || d.Replicas != nil || len(d.Labels) > 0 || d.UpdateConfig != nil || d.RestartPolicy != nil ||
```

Comment:
Suggestion: One possible way to make this less verbose is using `reflect.DeepEqual`, as in : 
```
import "reflect"

if !reflect.DeepEqual(d, DeployConfig{}) {
  // log the thing 
}
```